### PR TITLE
Translate Reconciliation 

### PR DESCRIPTION
**Pull request by mattveraldi**
Ciao, I finished the issue #71 .
On the _"Chiavi"_ paragraph, I chose to keep using `key` instead of translating it as 'chiave' because, in my opinion, it is more clear for the user that _we are specifically referring to the prop itself_, not only the concept. Let me know if it is ok for you to keep using 'key'. If that is the case, maybe we could consider to add it in the [Glossary](https://github.com/reactjs/it.reactjs.org/blob/master/GLOSSARY.md). Let me know if I should fix something on the translation or on the sentences and I will do it asap 😄 👍

### CODE REVIEW

**Comment by mattveraldi** [comment]
Any news so far?

**Comment by deblasis** [comment]
> Any news so far?

Ciao @mattveraldi!
I agree with keeping `key` untranslated because it's a **key concept**, pun intended 😄 

There's a merge conflict, as soon as you fix it I'll review so that we can merge.

File: content/docs/reconciliation.md [hunk 1]
```diff
@@ -141,30 +141,30 @@ In order to solve this issue, React supports a `key` attribute. When children ha
...
 
-1. The algorithm will not try to match subtrees of different component types. If you see yourself alternating between two component types with very similar output, you may want to make it the same type. In practice, we haven't found this to be an issue.
+1. L'algoritmo non cercherà di confrontare sottoalberi di tipi di componente diverso. Se il tuo caso è quello di alternare tra due componenti con diverso tipo ma con output molto simile, potresti renderli dello stesso tipo. in pratica, non abbiamo trovato problemi nel farlo.
 
-2. Keys should be stable, predictable, and unique. Unstable keys (like those produced by `Math.random()`) will cause many component instances and DOM nodes to be unnecessarily recreated, which can cause performance degradation and lost state in child components.
+2. Le chiavi dovrebbero essere stabili, predicibili ed univoche. Chiavi non stabili (come quell prodotte da `Math.random()`) causeranno la ricreazione unutile di diverse istanze dei componenti e nodi DOM, ciò può causare degradazione delle prestazioni e perdita di state nei componenti figlio.
```
Comment (deblasis) [inline content/docs/reconciliation.md:170]:
```suggestion
2. Le chiavi dovrebbero essere stabili, predicibili ed univoche. Chiavi non stabili (come quell prodotte da `Math.random()`) causeranno l'inutile ricreazione di diverse istanze dei componenti e nodi DOM, ciò può causare degradazione delle prestazioni e perdita di state nei componenti figlio.
```

**Review by deblasis — CHANGES_REQUESTED** [review]
Great job!

**Comment by mattveraldi** [comment]
> Great job!

Thank you 😄

**Review by deblasis — APPROVED** [review]

**Pull request merged**